source/common/redis/command_splitter_impl.cc

```diff
@@ -164,7 +164,9 @@ SplitRequestPtr InstanceImpl::makeRequest(const RespValue& request, SplitCallbac
     }
   }
 
-  auto handler = command_map_.find(request.asArray()[0].asString());
+  std::string to_lower_string(request.asArray()[0].asString());
+  to_lower_table_.toLowerCase(to_lower_string);
```

Comment:
Table is faster, and in these 2 cases, will matter, especially when eventually combined with lookup trie.